.circleci/config.yml

```diff
@@ -0,0 +1,65 @@
+defaults: &defaults
+  working_directory: ~/api-docs
+  docker:
+    - image: circleci/ruby:2.4.1-node-browsers
+      environment:
+        BUNDLE_JOBS: 3
+        BUNDLE_RETRY: 3
+        BUNDLE_PATH: vendor/bundle
+version: 2
+jobs:
+  install:
+    <<: *defaults
+    steps:
+      - checkout
+      # Which version of bundler?
+      - run:
+          name: Which bundler?
+          command: bundle -v
```

Comment:
This is test purpose?

Reply:
yes

Reply:
just to help to debug in case something goes wrong